web/src/main/webapp/table.ts

```diff
@@ -626,45 +628,53 @@ export class TableView extends RemoteTableObjectView
         }
     }
 
+    private hLogLog(colName: string): void {
+	let rr = this.createHLogLogRequest(colName);
+        rr.invoke(new HLogLogReceiver(this.getPage(), rr, "HLogLog",
+				      (res) => alert("Distinct values in column \'" +
+						     colName + "\' \u2248 " +
+						     String(res.distinctItemCount))));
+    }
+	
     private pca(): void {
         let colNames: string[] = [];
         this.selectedColumns.forEach(col => colNames.push(col));
-
+	
```

Comment:
Do not use spaces at the end of a line.
I can give you an emacs program for that.
I hope you are not using tabs.